### PR TITLE
add conv3d_trans_cudnn_op

### PR DESCRIPTION
**Pull request by chengduoZH**
fix https://github.com/PaddlePaddle/Paddle/issues/5617

### CODE REVIEW

File: paddle/operators/conv_transpose_cudnn_op.cc [hunk 1]
```diff
@@ -48,3 +65,14 @@ REGISTER_OP_CPU_KERNEL(
...
+REGISTER_OP(conv3d_transpose_cudnn, ops::ConvTransposeOp,
+            ops::CudnnConv3DTransposeOpMaker, conv3d_transpose_cudnn_grad,
+            ops::ConvTransposeOpGrad);
+
+REGISTER_OP_CPU_KERNEL(
+    conv3d_transpose_cudnn,
+    ops::GemmConvTransposeKernel<paddle::platform::CPUPlace, float>);
+REGISTER_OP_CPU_KERNEL(
+    conv3d_transpose_cudnn_grad,
+    ops::GemmConvTransposeGradKernel<paddle::platform::CPUPlace, float>);
```
Comment (qingqing01) [inline paddle/operators/conv_transpose_cudnn_op.cc]: REGISTER double for these ops.

Comment (chengduoZH) [reply paddle/operators/conv_transpose_cudnn_op.cc]: Done

File: paddle/operators/conv_transpose_cudnn_op.cu.cc [hunk 2]
```diff
@@ -234,3 +238,8 @@ REGISTER_OP_GPU_KERNEL(conv2d_transpose_cudnn,
                        ops::CudnnConvTransposeOpKernel<float>);
 REGISTER_OP_GPU_KERNEL(conv2d_transpose_cudnn_grad,
                        ops::CudnnConvTransposeGradOpKernel<float>);
+
+REGISTER_OP_GPU_KERNEL(conv3d_transpose_cudnn,
+                       ops::CudnnConvTransposeOpKernel<float>);
+REGISTER_OP_GPU_KERNEL(conv3d_transpose_cudnn_grad,
+                       ops::CudnnConvTransposeGradOpKernel<float>);
```
Comment (qingqing01) [inline paddle/operators/conv_transpose_cudnn_op.cu.cc]: There is no double for these ops. Please add in next PR.

Comment (chengduoZH) [reply paddle/operators/conv_transpose_cudnn_op.cu.cc]: Ok.

**Pull request merged**